lib/improver/tests/psychrometric_calculations/psychrometric_calculations/test_PhaseChangeLevel.py

```diff
@@ -28,29 +28,71 @@
 # CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
-"""Unit tests for psychrometric_calculations FallingSnowLevel."""
+"""Unit tests for psychrometric_calculations PhaseChangeLevel."""
+
 import unittest
 
-import iris
 import numpy as np
 from cf_units import Unit
+import iris
 from iris.tests import IrisTest
 
 from improver.psychrometric_calculations.psychrometric_calculations import (
-    FallingSnowLevel)
+    PhaseChangeLevel)
+from improver.utilities.cube_manipulation import sort_coord_in_cube
 from improver.tests.set_up_test_cubes import (set_up_variable_cube,
                                               add_coordinate)
 
 
+class Test__init__(IrisTest):
+
+    """Test the init method."""
+
+    def test_snow_sleet(self):
+        """Test that the __init__ method configures the plugin as expected
+        for the snow-sleet phase change."""
+
+        phase_change = 'snow-sleet'
+        plugin = PhaseChangeLevel(phase_change, grid_point_radius=3)
+
+        self.assertEqual(plugin.falling_level_threshold, 90.)
+        self.assertEqual(plugin.phase_change_name, 'snow_falling')
+        self.assertEqual(plugin.missing_data, -300.)
+        self.assertEqual(plugin.grid_point_radius, 3)
+
+    def test_sleet_rain(self):
+        """Test that the __init__ method configures the plugin as expected
+        for the sleet_rain phase change."""
+
+        phase_change = 'sleet-rain'
+        plugin = PhaseChangeLevel(phase_change, grid_point_radius=3)
+
+        self.assertEqual(plugin.falling_level_threshold, 202.5)
+        self.assertEqual(plugin.phase_change_name, 'rain_falling')
+        self.assertEqual(plugin.missing_data, -300.)
+        self.assertEqual(plugin.grid_point_radius, 3)
+
+    def test_unknown_phase_change(self):
+        """Test that the __init__ method raised an exception for an unknown
+        phase change argument."""
+
+        phase_change = 'kittens-puppies'
```

Comment:
You could have been more pythonic with "europeanswallow-africanswallow"!

Reply:
I have to include Kittens in case Caroline S reviews this. We always include kittens somewhere.